contracts/ccip/ccip/sources/fee_quoter.move

```diff
@@ -568,7 +568,7 @@ public fun update_prices_with_owner_cap(
     let fee_quoter_cap = new_fee_quoter_cap(owner_cap, ctx);
     update_prices(
         ref,
-        &fee_quoter_cap,
+        // &fee_quoter_cap,
```

Comment:
instead of commenting these out, we can use [this](https://github.com/smartcontractkit/chainlink-sui/blob/develop/contracts/ccip/ccip/sources/fee_quoter.move#L558-L567) to update prices from owner cap. or we can create a fee quoter cap with this [function](https://github.com/smartcontractkit/chainlink-sui/blob/develop/contracts/ccip/ccip/sources/fee_quoter.move#L278-L282) to update prices and then destroy it with [this](https://github.com/smartcontractkit/chainlink-sui/blob/develop/contracts/ccip/ccip/sources/fee_quoter.move#L1863-L1866).
i will be adding op support for these functions now.